lib/src/components/modal/modal.scss

```diff
@@ -42,6 +42,7 @@ $modal-small-width: 400px !default;
     0
   ) !important; //Should be theme("colors.neutral.20") but it doesn't work for some reason and have spent too long on this!
   @apply flex justify-center items-center transition ease-out duration-300;
+  z-index: 501;
```

Comment:
Ooft this... the `Header/TopBar` has a z-index of 500 for some reason. Meaning the overlay of the new modal wasn't on top. The highest z-index for tailwind is 50, so this is hard coded instead of the helper.